picard/ui/mainwindow.py

```diff
@@ -113,8 +121,9 @@ def __init__(self, parent=None):
         self.selected_objects = []
         self.ignore_selection_changes = False
         self.toolbar = None
-        self.player = QtMultimedia.QMediaPlayer(self)
-        self.player.error.connect(self._on_player_error)
+        if qt_multimedia_available:
```

Comment:
We could also try to import directly here and only set player property if a player is actually available.

```python
self.player = None
try:
    from PyQt5 import QtMultimedia
    player = QtMultimedia.QMediaPlayer(self)
    if player.availability() != QtMultimedia.QMultimedia.ServiceMissing:
        self.player = player
        self.player.error.connect(self._on_player_error)
except ImportError as e:
    log.warning('Internal player disabled: %s', e.msg)
```

It would simplify tests:

```python
if self.player:
   do_something_with(self.player)
```

And ensure an exception is raised if `self.player` is used without being really available (like using `self.player.volume()`).

@timur-enikeev @phw : what do you think ?

Reply:
I am not a big fan of importing inline, I think this is clearer if done at the top. Apart from that I agree, so I would change your example to:

```python
self.player = None
player = QtMultimedia.QMediaPlayer(self)
if qt_multimedia_available and player.availability() != QtMultimedia.QMultimedia.ServiceMissing:
    self.player = player
    self.player.error.connect(self._on_player_error)
```

Reply:
I implemented this, with extra error logging in https://github.com/metabrainz/picard/pull/1207